python_modules/dagster/dagster/_core/definitions/asset_selection.py

```diff
@@ -200,6 +201,28 @@ def key_prefixes(
             selected_key_prefixes=_asset_key_prefixes, include_sources=include_sources
         )
 
+    @public
+    @staticmethod
+    def key_subsets(
+        *key_subsets: CoercibleToAssetKeySubset, include_sources: bool = False
```

Comment:
Hmm, I see that this mirrors the key prefix constructor, but IMO would be easier to use if it just took a single key subset, since I think that's the most common use-case and the one we'd be using with the new key selection syntax, right?